tests/unit/records/test_records_receivers.py

```diff
@@ -1066,3 +1067,44 @@ def test_populate_author_count_does_nothing_if_record_is_not_literature():
     populate_author_count(None, record)
 
     assert 'author_count' not in record
+
+
+def test_populate_populate_authors_full_name_normalized():
+    schema = load_schema('hep')
+    subschema = schema['properties']['authors']
+
+    record = {
+        '$schema': 'http://localhost:5000/records/schemas/hep.json',
+        'authors': [
+            {
+                'full_name': 'Vurës, John',
+            },
+            {
+                'full_name': 'Vures, John',
+            },
+        ],
+    }
+    assert validate(record['authors'], subschema) is None
```

Comment:
IIRC @jacquerie told me that these assertions are not needed anymore in receiver tests.

Reply:
No, those are good assertions to have, as they protect the tested code against changes in the schema. What were bad were the assertions on the `$schema` property, which I removed in https://github.com/inspirehep/inspire-next/commit/08231012d665677c84c7e4ef4a9e34dc55feb15d#diff-d73f938cbaa6c2c71bc00274ac594f14.